lib/engine/game/g_18_carolinas/game.rb

```diff
@@ -297,14 +343,62 @@ def operating_round(round_num)
           Round::Operating.new(self, [
             Engine::Step::Bankrupt,
             Engine::Step::HomeToken,
-            Engine::Step::Track,
+            G18Carolinas::Step::Track,
             Engine::Step::Token,
             Engine::Step::Route,
             Engine::Step::Dividend,
             Engine::Step::BuyTrain,
           ], round_num: round_num)
         end
 
+        def upgrades_to?(from, to, special = false, selected_company: nil)
+          standard = to.paths.any? { |p| p.track == :broad }
```

Comment:
partition

Reply:
OK

Reply:
Hmmm. Except standard and southern are booleans, not lists of paths. standard needs to be true if any path has broad track. southern needs to be true if any path isn't broad.